pkg/util/http_utils.go

```diff
@@ -0,0 +1,42 @@
+package util
+
+import (
+	"net"
+	"net/http"
+	"time"
+)
+
+// currently mirrors the default http.Transport values
+var jxDefaultTransport http.RoundTripper = &http.Transport{
+	DialContext: (&net.Dialer{
+		Timeout:   30 * time.Second,
+		KeepAlive: 30 * time.Second,
+		DualStack: true,
+	}).DialContext,
+	MaxIdleConns:          100,
+	IdleConnTimeout:       90 * time.Second,
+	TLSHandshakeTimeout:   10 * time.Second,
+	ExpectContinueTimeout: 1 * time.Second,
```

Comment:
can we make these configurable with env vars maybe, using the values above as defaults?

Reply:
sure thing